ci_framework/roles/edpm_deploy_baremetal/defaults/main.yml

```diff
@@ -27,3 +27,4 @@ cifmw_edpm_deploy_baremetal_wait_bmh_timeout_mins: 20
 cifmw_edpm_deploy_baremetal_wait_dataplane_timeout_mins: 30
 cifmw_edpm_deploy_baremetal_update_os_containers: false
 cifmw_edpm_deploy_baremetal_repo_setup_override: false
+cifmw_edpm_deploy_baremetal_using_bmaas: true
```

Comment:
is it in the README of the role?

Reply:
No, I had missed updating the `readme`. Thank you, I have updated the same in the new commit